@plotly/dash-test-components/src/components/MyPersistedComponentNested.js

```diff
@@ -0,0 +1,298 @@
+import React, {PureComponent} from 'react';
```

Comment:
I think this can be 🔪 altogether. Nested vs. non-nested components have to do with the generator only.